src/VirtoCommerce.CustomerModule.Web/Scripts/blades/customer-detail.tpl.html

```diff
@@ -64,6 +64,15 @@
                         </ui-select-choices>
                     </ui-select>
                 </div>
+                <div class="form-group">
+                    <label class="form-label">{{'customer.blades.contact-detail.labels.associated-organizations' | translate}}</label>
+                    <ui-select multiple ng-model="blade.currentEntity.associatedOrganizations">
+                        <ui-select-match placeholder="{{'customer.blades.contact-detail.placeholders.associated-organizations' | translate}}">{{$item.name}}</ui-select-match>
+                        <ui-select-choices repeat="x.id as x in organizations" refresh="fetchOrganizations($select)" when-scrolled="fetchNextOrganizations($select)">
```

Comment:
Probably we shouldn't use the same `organizations` list for both combos to avoid flickering. need to have 2 different lists. 
The similar problem with flickering and multiple requests was here: https://github.com/VirtoCommerce/vc-module-store/commit/b1ef231bec9f67ad38d60c1388970cfa03840c15

Reply:
Fixed by using separate properties